src/Traits/DealsWithMigrations.php

```diff
@@ -12,6 +12,6 @@ protected function getMigrationPaths()
             return parent::getMigrationPaths();
         }
 
-        return [config('tenancy.migrations_directory', database_path('migrations/tenant'))];
+        return config('tenancy.migrations_paths', [database_path('migrations/tenant')]);
```

Comment:
Fallback to previous config to maintain backwards compatibility:

```suggestion
        return config('tenancy.migrations_paths', [config('tenancy.migrations_directory') ?? database_path('migrations/tenant')]);
```

Reply:
Please commit this suggestion.

Reply:
When you commit this, I can merge the pull request.